tests/GameVersionTests.cpp

```diff
@@ -267,4 +267,19 @@ TEST(GameVersion_Tests, LessThanOrEqualsFromEquals)
 	const GameVersion versionTwo(1, 2, 3, 4);
 
 	ASSERT_LE(versionTwo, version);
+}
+
+TEST(GameVersion_Tests, VersionEqual)
+{
+	const GameVersion version("1.3.3.0");
+	const GameVersion requiredVersion("1.3.3");
+	ASSERT_EQ(version, requiredVersion);
+	ASSERT_TRUE(version == requiredVersion);
```

Comment:
Redundant, the ASSERT_EQ lin already demonstrated this.

Reply:
I wanted to additionally check if the comparison operator works as intended for GameVersion but maybe it's outside the scope of the test.

Reply:
This check is preformed implicitly by the first ASSERT_EQ command. It uses whatever == is to check for that.